scene/3d/soft_body_3d.cpp

```diff
@@ -215,7 +215,13 @@ bool SoftBody3D::_set_property_pinned_points_attachment(int p_item, const String
 
 	if ("spatial_attachment_path" == p_what) {
 		PinnedPoint *w = pinned_points.ptrw();
-		callable_mp(this, &SoftBody3D::_pin_point_deferred).call_deferred(Variant(w[p_item].point_index), true, p_value);
+
+		if (is_inside_tree()) {
+			callable_mp(this, &SoftBody3D::_pin_point_deferred).call_deferred(Variant(w[p_item].point_index), true, p_value);
+		} else {
+			pin_point(Variant(w[p_item].point_index), true, p_value);
```

Comment:
Adding `Variant(...)` is unnecessary, it can be just `w[p_item].point_index`.

Reply:
I amended the commit to use just `w[p_item].point_index`.